package.json

```diff
@@ -39,6 +39,7 @@
     "express-session": "^1.17.3",
     "helmet": "^7.0.0",
     "ioredis": "^5.3.2",
+    "jest-offline": "^1.0.1",
```

Comment:
Is this needed? the dependency `jest-offline` is not actively maintained (8 years old).